tests/test-performance.js

```diff
@@ -0,0 +1,26 @@
+var performance = require("performance");
+
+var total = 0;
+var passed = 0;
+
+function assert(actual, description) {
+    total += 1;
+    var label = "\033[1m\033[31mFAIL\033[0m";
+    if (actual === true) {
+        passed += 1;
+        label = "\033[1m\033[32mPASS\033[0m";
+    }
+    console.log(label + " - " + description);
+}
+
+var before = performance.now();
+//console.log("performance.now() before:", before);
+
+setTimeout(function() {
+    var after = performance.now();
+    //console.log("performance.now() after:", after);
+    var diff = after - before;
+    // Allow for some jitter, especially on Linux
+    assert(diff > 990 && diff < 1010, "performance.now() result over known delay");
```

Comment:
A101 also has 10ms timer resolution, so maybe you meant to use >= and <= here? When I see before = 30 and after = 1040, I get a failure.

Other than that +1, I'll see what you say and merge it tomorrow.